### ISSUE TITLE
Shape up the basic usage examples

### ISSUE THREAD

**Issue by aqrln** [metarhia/JSTP#131]
Is anyone willing to write a couple of usage examples (something completely basic like a server with one method and a client that calls it to begin with, and a bit more complex apps later)? The `examples` directory is where they are supposed to reside in.

@nechaido would you be interested in doing that?
/cc @belochub

**Comment by nechaido** [comment]
@aqrln yes, I am. But I think we`ll need tests first. Also good written test is an example itself, we can use them as examples if we remove asserts and other testing stuff. Am I right?

**Comment by aqrln** [comment]
@nechaido good written tests are examples to some extent, but examples that are supposed to be examples are better examples ;)

But that's okay, if you want to dedicate your time to tests first, then go for it, that would be great 🎉
